### ISSUE TITLE
Fix automatic ARM cross-compilation on travis

### ISSUE THREAD

**Issue by etr** [etr/libhttpserver#131]
**Is your feature request related to a problem? Please describe.**
Many users often report building the library for ARM. This would prevent problems in the building system for these use-cases.

**Describe why the feature or enhancement you are proposing fits the library.**
Given the library is used quite a lot on embedded systems, this fits the use-case by delivering certainty of a successful build.

**Describe the solution you'd like**
Include a cross-compilation test-case in travis.

**Describe alternatives you've considered**
None

**Comment by LeSpocky** [comment]
Travis was removed with #222, but I guess the feature is still wanted, but for github actions now?

**Comment by etr** [comment]
yep. We still don't have a system to perform automated cross-compilation on ARM over the current CI tool (github-actions). Renamed the issue.
